packages/checklist/lib/utils/extensions/extensions.dart

```diff
@@ -0,0 +1,65 @@
+// Importing necessary packages and files
+import 'package:digit_data_model/data_model.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:intl/intl.dart';
+import '../utils.dart';
+
+// Importing the file 'context_utility.dart' with a 'part' directive.
+part 'context_utility.dart';
+
+// Extension to calculate age based on a DateTime object.
+extension DateAgeExtension on DateTime {
+  int get age {
```

Comment:
> Please check if this is being used else delete this. Since age is not used in checklist.

We are deleting the extensions file as it is not being used anywhere